src/main/java/org/cryptomator/common/locationpresets/GoogleDriveMacLocationPresetsProvider.java

```diff
@@ -0,0 +1,107 @@
+package org.cryptomator.common.locationpresets;
+
+import org.cryptomator.integrations.common.CheckAvailability;
+import org.cryptomator.integrations.common.OperatingSystem;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+import static org.cryptomator.integrations.common.OperatingSystem.Value.MAC;
+
+@OperatingSystem(MAC)
+public final class GoogleDriveMacLocationPresetsProvider implements LocationPresetsProvider {
+	private static final Path LOCATION = LocationPresetsProvider.resolveLocation("~/Library/CloudStorage/").toAbsolutePath();
```

Comment:
I am not a big fan of the name `LOCATION` since it's actually the parent, maybe think of something like `ROOT_LOCATION`.